static/app/components/noProjectMessage.spec.tsx

```diff
@@ -42,10 +43,38 @@ describe('NoProjectMessage', function () {
     expect(screen.getByRole('button', {name: 'Create project'})).toBeEnabled();
   });
 
-  it('disable "Create Project" when user has no org-level access', function () {
+  it('enable "Create Project" when user is team admin', function () {
     ProjectsStore.loadInitialData([]);
+    TeamStore.loadInitialData([
+      TeamFixture({
+        id: '1',
+        slug: 'team1',
+        name: 'Team 1',
+        isMember: true,
+        teamRole: 'admin',
+        access: ['team:admin', 'team:write', 'team:read'],
+      }),
+    ]);
+
+    render(<NoProjectMessage organization={noAccessOrg} />, {organization: noAccessOrg});
+
+    expect(screen.getByRole('button', {name: 'Create project'})).toBeEnabled();
+  });
+
+  it('disable "Create Project" when user is not team admin', function () {
```

Comment:
Might be good to add a test for when `allowMemberProjectCreation` is disabled as well